spec/rest/logger.test.js

```diff
@@ -0,0 +1,139 @@
+"use strict";
+
+define(['shared_helper'], function(helper) {
+	var exports = {};
+
+	var isBrowser = !!(typeof window !== 'undefined' && typeof navigator !== 'undefined' && window.document);
+
+	/* If we are running node, we mimic the loading mechanism used in /nodejs/realtime.js and /nodejs/rest.js
+	 * to get a reference to the logger. We do this because its implementation is not a module.
+	 * 
+	 * If we are in a browser, the logger is already loaded manually via a script tag in nodeunit.html.
+	 */
+
+	var logger;
+	
+	if (isBrowser) {
+		logger = Logger;
+	} else {
+
+		var
+			fs   = require('fs'),
```

Comment:
I'm mainly asking because I like to set Eclipse to fix these things automatically on file save. Then I don't have to waste time on the trivial stuff. I currently have my tab set to 4 characters.

In the last project I worked on we followed Drupal coding standards, and there are publicly available settings that you can download and import into your favourite IDE. Perhaps we could do something similar. I can have a look later.